src/paths/bellman_ford.c

```diff
@@ -203,3 +210,285 @@ int igraph_shortest_paths_bellman_ford(const igraph_t *graph,
 
     return 0;
 }
+
+
+/**
+ * \ingroup structural
+ * \function igraph_get_shortest_paths_bellman_ford
+ * \brief Calculates the weighted shortest paths from/to one vertex 
```

Comment:
Not sure if our doc parser can parse multi-line `\brief` directives; please write it on one line and add a comma before "allowing"